packages/cloud-cognitive/src/components/SidePanel/_side-panel.scss

```diff
@@ -93,14 +93,16 @@ $max-panel-size: 75%; // set max-width on max panels to 75%
   }
 
   .#{$block-class}__container {
+    --#{$block-class}--subtitle-opacity: 1;
+    --#{$block-class}--title-font-size: 1.25rem;
```

Comment:
Should we be getting this value from a carbon token?

Reply:
Ideally yes, but the font size for the title is now controlled by this css variable which gives us a smoother title animation. And I can't set the default value of this css variable to a sass variable.

ie this does not work unfortunately

```scss
--#{$block-class}--title-font-size: $productive-heading-03;
```